tensorflow_addons/optimizers/conditional_gradient.py

```diff
@@ -182,13 +178,9 @@ def _resource_apply_dense(self, grad, var, apply_state=None):
             )
             s = top_singular_vector
 
-        var_update_tensor = tf.math.multiply(var, lr) - (1 - lr) * lambda_ * s
-        var_update_kwargs = {
-            "resource": var.handle,
-            "value": var_update_tensor,
-        }
-        var_update_op = tf.raw_ops.AssignVariableOp(**var_update_kwargs)
-        return tf.group(var_update_op)
+        return var.assign(
+            var * lr - (1 - lr) * lambda_ * s, use_locking=self._use_locking
```

Comment:
How about creating a temp variable for the expression? 
```python
var_update = xxxx
var.assign(xxx)
```

Reply:
Done :-)